eZ/Publish/Core/Repository/Permission/PermissionResolver.php

```diff
@@ -241,6 +253,126 @@ public function canUser($module, $function, ValueObject $object, array $targets
         return false; // None of the limitation sets wanted to let you in, sorry!
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function lookupLimitations(
+        string $module,
+        string $function,
+        ValueObject $object,
+        array $targets = [],
+        array $limitationsIdentifiers = []
+    ): LookupLimitationResult {
+        $permissionSets = $this->hasAccess($module, $function);
+
+        if ($permissionSets === false || $permissionSets === true) {
```

Comment:
is_bool?

Reply:
Hmm, this is copy-paste from `canUser`, but maybe it's a good idea to go for something less strange ;) 
Fixed in c67e9d3.